docs/vnext/napa-versions.md

```diff
@@ -0,0 +1,22 @@
+# Napa.js versions
+
+
+There are 2 versions of Napa.js:
+
+- v0.2.x (Current): This is the current release version.
+
+- v0.4.x (vNext): This is a version with new worker implementation.
+
+## Current
+
+Current is the stable release version. It's the `master` branch in Napa.js repo.
+
+## vNext
+
+vNext stands for the next release version. Version will be rolled from canary -> pre -> release.
+
+- v0.4.x-canary: working version with main function ready. Need to work with a patched node.
```

Comment:
Good statement about the 3 stage idea!
But instead of a given special name, I would prefer a name with some meaning. What do you think of v0.4.x-**peek**? I am okay with any other words that is more accurate.